# HG changeset patch
# User test
# Date 0 0
#      Thu Jan 01 00:00:00 1970 +0000
# Node ID 50037f45bff0c6ab91ecbd640de55cc153f771c9
# Parent  0a9df329a4fe6fa719ca330eda9fbf7bf8573115
display the relative date in pushlog

ie makes the date appear with "60 minutes ago" text

diff -r 0a9df329a4fe -r 50037f45bff0 hgtemplates/gitweb_mozilla/changeset.tmpl
--- a/hgtemplates/gitweb_mozilla/changeset.tmpl	Thu Jan 01 00:00:00 1970 +0000
+++ b/hgtemplates/gitweb_mozilla/changeset.tmpl	Thu Jan 01 00:00:00 1970 +0000
@@ -54,7 +54,7 @@
 {child%changesetchild}
 <tr><td>push id</td><td>{if(pushid, '<a href="{url|urlescape}pushloghtml?changeset={node}">{pushid}</a>', 'unknown')}</td></tr>
 <tr><td>push user</td><td>{if(pushuser, pushuser|escape, 'unknown')}</td></tr>
-<tr><td>push date</td><td>{if(pushdate, pushdate|rfc822date, 'unknown')}</td></tr>
+<tr><td>push date</td><td class="date age">{if(pushdate, pushdate|rfc822date, 'unknown')}</td></tr>
 {if(convertsourcepath, '<tr><td>converted from</td><td><a href="{convertsourcepath}/rev/{convertsourcenode}">{convertsourcenode}</a></td></tr>')}
 {if(treeherderrepourl, if(pushhead, '<tr><td>treeherder</td><td>{treeherderrepo|escape}@{pushhead|short} [<a href="{treeherderrepourl}&revision={pushhead}">default view</a>] [<a href="{treeherderrepourl}&revision={pushhead}&filter-resultStatus=testfailed&filter-resultStatus=busted&filter-resultStatus=exception">failures only]</td></tr>'))}
 {if(git_repo_url, if(startswith("tags-",branch), '', '<tr><td>git commit</td><td><a href="{git_repo_url}/commit/{git_commit}" target="_blank">{git_commit}</a></td></tr>'))}
